packages/gallery/src/components/gallery/proGallery/galleryContainer.js

```diff
@@ -637,6 +638,17 @@ export class GalleryContainer extends React.Component {
     }
   }
 
+  getAriaAttributes() {
```

Comment:
Please remove this function outside of the class (and from this file)
Remove the dependency of `this` from the function. 
Accept the relevant props as argument